ecosystem/typescript/sdk_v2/src/types/hex.ts

```diff
@@ -0,0 +1,91 @@
+import { bytesToHex, hexToBytes } from "@noble/hashes/utils";
+import { HexInput } from ".";
```

Comment:
> but I think in general it would be better to put the types in a different files, maybe types.ts

oh I see that `hex.ts` is already under a `types` folder.
At least to me, a type in Typescript is either an `interface` or a `type`, so maybe we could move `Hex` to a `utils` folder since it's a utility class?

Reply:
@0xmaayan what do you think? I don't really mind but to me utils is just assorted utility functions, not a place for core "types" (classes in this case) that we use throughout the whole SDK.

Reply:
We can also just move `HexInput` under `hex.ts` :D

Reply:
yes the idea is to have all types or interfaces on a types.ts file
```
types // holds local types
```

Let's move this class into utils

Reply:
yes the idea is to have all types and interfaces under `types.ts`
```
types // holds local types
```

let's have this class under `utils` folder

Reply:
Spoke offline, we'll do `core`.